src/Raven.Server/Documents/Sharding/Queries/ShardedQueryProcessor.cs

```diff
@@ -165,8 +165,12 @@ private void RewriteQueryForPaging(ref BlittableJsonReaderObject queryTemplate)
         var limit = ((_query.Limit ?? 0) + (_query.Offset ?? 0)) * (long)_parent.DatabaseContext.ShardCount;
         if (limit > int.MaxValue) // overflow
             limit = int.MaxValue;
+
         modifiedArgs[limitToken] = limit;
 
+        queryTemplate.Modifications.Remove(nameof(IndexQueryServerSide.Start));
+        queryTemplate.Modifications.Remove(nameof(IndexQueryServerSide.PageSize));
```

Comment:
Those can be sent by older clients, right?

Reply:
It is sent by older clients and is translated correctly into RQL.
Than `Start` and `PageSize` are used in `IndexQueryServerSide`.
When we send it again to the shards we need to remove it.